Enterprise-CMCS/eAPD
# kong changes

Resolves #4539 

### Description
Updates the authentication for the endpoints. Removed the LaunchDarkly flag. Added a check for apikey. The rest of the auth is being handled through the WAF. Changed the patch endpoint to be a post, so that it was clearer.

### This pull request is ready to code review when

      tests, refactor

### Steps to manually verify this change
1. open postman
2. Check Good GET
	1. set postman to GET https://ec2-54-147-193-218.compute-1.amazonaws.com/api/apds/submissions
	2. under Headers add a row for `apikey` and set it to any value
	3. click Send
	4. expect an error Could not get response
	5. click that link to Disable SSL Verification under that response
	6. click Send
	7. expect to get a 200 response with an empty array
3. Check Bad Auth GET 
	1. uncheck the apikey row
	2. click Send
	3. expect to get a 403 response
4. Check Good POST
	1. change GET to POST and check the apikey row
	2. under Body select raw and JSON and paste `[{ "apdId": "apd id", "newStatus": "completed" }]` in the text area
	3. click Send
	4. expect a 200 response with a body of `[{"apdId":"apd id","success":false,"error":"APD Id is invalid"}]`
8. Check Bad Request POST
	1. remove value from text area
	2. click Send
	3. expect a 400 response
9. Check Bad Auth POST
	1. uncheck the apikey row
	2. click Send
	3. expect a 403 response

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
